src/executables/unload_schema.c

```diff
@@ -2399,12 +2399,26 @@ emit_attribute_def (print_output & output_ctx, DB_ATTRIBUTE * attribute, ATTRIBU
     {
     case INSTANCE_ATTRIBUTE:
       {
-	output_ctx ("       %s%s%s ", PRINT_IDENTIFIER (name));
+	if (strchr (name, ']') != NULL)
+	  {
+	    output_ctx ("       %s%s%s ", PRINT_IDENTIFIER_WITH_QUOTE (name));
+	  }
+	else
+	  {
+	    output_ctx ("       %s%s%s ", PRINT_IDENTIFIER (name));
+	  }
```

Comment:
All three cases seem to do the same thing.
How about changing it like below?

```

case CLASS_ATTRIBUTE:
         /*
          * NOTE: The parser no longer recognizes a CLASS prefix for class
          * attributes, this will have been encoded in the surrounding
          * "ADD CLASS ATTRIBUTE" clause
          */
case INSTANCE_ATTRIBUTE:
case SHARED_ATTRIBUTE:
         if (strchr (name, ']') != NULL)
         {
               output_ctx(" %s%s%s ", PRINT_IDENTIFIER_WITH_QUOTE (name)); 
          }
        else
          { 
                output_ctx(" %s%s%s ", PRINT_IDENTIFIER (name));
           }
      break;
```

Reply:
I will change it like this. Thank you.